source/Heuristics.h

```diff
@@ -155,6 +253,26 @@ class Heuristics {
    * This is also the maximum collapse depth for inferred propagations.
    */
   constexpr static std::uint32_t kPropagationMaxCollapseDepth = 4;
+
+private:
+  std::size_t k_join_override_threshold_;
+  std::size_t k_android_join_override_threshold_;
+  std::optional<std::size_t> k_warn_override_threshold_;
+  std::size_t k_generation_max_port_size_;
+  std::size_t k_generation_max_output_path_leaves_;
+  std::size_t k_parameter_source_max_port_size_;
+  std::size_t k_parameter_source_max_output_path_leaves_;
+  std::size_t k_sink_max_port_size_;
+  std::size_t k_sink_max_input_path_leaves_;
+  std::size_t k_call_effect_source_max_port_size_;
+  std::size_t k_call_effect_source_max_output_path_leaves_;
+  std::size_t k_call_effect_sink_max_port_size_;
+  std::size_t k_call_effect_sink_max_input_path_leaves_;
+  std::size_t k_max_number_iterations_;
+  std::size_t k_max_depth_class_properties_;
+  std::size_t k_max_call_chain_source_sink_distance_;
+  std::size_t k_propagation_max_input_path_size_;
+  std::size_t k_propagation_max_input_path_leaves_;
```

Comment:
nit: I think the `k_` prefix should be removed... it's meant to denote constants and these private members aren't really constants anymore.

Same applies to the `set_k_<heuristic>()` methods... they could just be `set_<heuristic>()`

(the `constexpr` stuff above are still constants though)

Reply:
Solved in bb70ecff9a36b20fd217d8c3afd8b2466505c418